Source/GUI/dvrescue/dvrescue/main.qml

```diff
@@ -81,6 +92,14 @@ ApplicationWindow {
             }
             icon.source: "/icons/menu-debug.svg"
         }
+
+        NavButton {
+            // text: qsTr("Debug")
+            onClicked: {
+                about.open()
+            }
+            icon.source: "/icons/menu-debug.svg"
```

Comment:
I see the page is accessed via a copy/paste of the "DEBUG" button. I suggest removing it from the sidecar altogether and instead have it in the toolbar, such as in "dvrescue > About dvrescue", similar to how it's accessed in QCTools.